src/installation/index.md

```diff
@@ -109,20 +111,23 @@ Once `espup` is installed, do the following:
     ```sh
     espup install
     ```
-
-    `espup` will create an export file that contains some environment variables required to build projects:
-
-    - Unix systems - `$HOME/export-esp.sh`
-    - Windows - `%USERPROFILE%\export-esp.ps1`
-
-2. On Unix systems, make sure to source this file in every terminal before building any application: `. $HOME/export-esp.sh`
-    > On Windows systems, no need to source the file. It is only created to show the modified environment variables.
-
-
-After running `espup install`:
-
-- `no_std` (bare-metal) applications should work out of the box
-- `std` applications require additional software covered in [`std` Development Requirements][rust-esp-book-std-requirements]
+2. Set up the environment variables. `espup` will create an export file that contains some environment variables required to build projects:
+    - Unix systems (`$HOME/export-esp.sh`). There are different ways of sourcing the file:
+      - Source this file in every terminal:
+        1. Source the export file: `. $HOME/export-esp.sh`
+        - This requires running the command in every new shell.
+      - Create an alias for executing the `export-esp.sh`:
+        1. Copy and paste the following command to your shell’s profile (`.profile`, `.bashrc`, `.zprofile`, etc.): `alias get_esprs='. $HOME/export-esp.sh'`
+        2. Refresh the configuration by restarting the terminal session or by running `source [path to profile]`, for example, `source ~/.bashrc`.
+        - This requires running the alias in every new shell.
```

Comment:
This looks like a mixed number / bullet-point list which is unusual. Suggest placing this line directly after the upper-level bullet point like shown below, because this information might also influence the readers decision about whether to choose this method.

```
- Create an alias for executing the `export-esp.sh`:<br>
  (requires running the alias in every new shell)
  1. Copy and paste the following ...
  2. ...
```